src/submitter.py

```diff
@@ -70,6 +75,19 @@ def override(self, **kwargs):
         for key, val in old_values.items():
             setattr(self, key, val)
 
+    def get_destinations(self):
```

Comment:
This makes it easy to support both configuration modes. In either case, the handler code just works with a list of `Destination` instances.